document/src/tests/documentupdatetestcase.cpp

```diff
@@ -550,61 +458,61 @@ void DocumentUpdateTest::testReadSerializedFile()
     DocumentUpdate& upd(*updp);
 
     const DocumentType *type = repo.getDocumentType("serializetest");
-    CPPUNIT_ASSERT_EQUAL(DocumentId(DocIdString("update", "test")), upd.getId());
-    CPPUNIT_ASSERT_EQUAL(*type, upd.getType());
+    EXPECT_EQ(DocumentId(DocIdString("update", "test")), upd.getId());
+    EXPECT_EQ(*type, upd.getType());
 
     // Verify assign value update.
     FieldUpdate serField = upd.getUpdates()[1];
-    CPPUNIT_ASSERT_EQUAL(serField.getField().getId(), type->getField("intfield").getId());
+    EXPECT_EQ(serField.getField().getId(), type->getField("intfield").getId());
 
     const ValueUpdate* serValue = &serField[0];
-    CPPUNIT_ASSERT_EQUAL(serValue->getType(), ValueUpdate::Assign);
+    EXPECT_EQ(serValue->getType(), ValueUpdate::Assign);
```

Comment:
Consider changing `getType()` checks in this test to `ASSERT_EQ`; blind `static_assert`s are done based on the checks

Reply:
Fixed